jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/jdbc/JdbcAdapterProperties.java

```diff
@@ -64,6 +66,12 @@ private static String getProperty(final Map<String, String> properties, final St
         }
     }
 
+    public static List<String> getIgnoreErrorList(final Map<String, String> properties) {
+        String ignoreErrors = getProperty(properties, PROP_IGNORE_ERROR_LIST, "");
+        List<String> ignoreErrorsList = Arrays.asList(ignoreErrors.split(","));
+        return ignoreErrorsList.stream().map(error -> error.trim().toUpperCase()).collect(Collectors.toList());
```

Comment:
Skip intermediate `List` creation and use fluent programming:
```java
return Arrays //
    .stream(ignoreErrors.split(",")) //
    .map(String::trim) //
    .map(String::toUpperCase) //
    .collect(Collectors.toList());

Reply:
done